src/main/java/nextstep/fp/StreamStudy.java

```diff
@@ -28,17 +27,27 @@ public static void printLongestWordTop100() throws IOException {
         List<String> words = Arrays.asList(contents.split("[\\P{L}]+"));
 
         // TODO 이 부분에 구현한다.
+        words.stream()
+             .distinct()
+             .filter(word -> word.length() > 12)
```

Comment:
```suggestion
             .filter(word -> word.length() > 12)
             .distinct()
```
[`중복제한을 filter() 이후에 하게 되면 어떻게될까요?`](https://github.com/next-step/java-ladder/pull/1234#discussion_r762497307)

라고 드렸던 부분은 filter() > distinct() 순의로 작업의 순서를 변경해보시면 좋을 것 같아 의견을 드린 부분이에요. :)

<img width="843" alt="스크린샷 2021-12-05 오후 3 29 03" src="https://user-images.githubusercontent.com/4317804/144736270-920e99d1-afaa-44de-9b9e-862b228192e4.png">

filter()와는 상관없이 이전에는 sorted()를 한 후에 distinct()가 실행되게 되어있었어요~